### ISSUE TITLE
【question】how to use this

### ISSUE THREAD

**Issue by QingLeiLi** [Force67/DawnHook#1]
I am new to mod, i find this thouth google, could you tell me how to use this dll? I have a `Ultimate-ASI-Loader`, replace the `dawnHook.dll` to the game dir, then? what should i do?

Thank you very much.

**Comment by Force67** [comment]
Yes you should put dawnhook together with the ultimate asi loader in the games directory. Keep in mind, that dawnhook needs to have the extension ".asi" in order to get recognized by the asi loader.

**Comment by QingLeiLi** [comment]
Yes, I know that, I mean what can dawnhook do in the game?

**Comment by Force67** [comment]
It features some basic virtual filesystem manipulation, and a lua executor with a ingame gui.

**Comment by QingLeiLi** [comment]
Thanks for the response，the dll work fine until I got a `untrusted system file` error, I know there should be a Easy Anti-Cheat patching, I set it with [this](https://fearlessrevolution.com/viewtopic.php?f=4&t=6334), then the dll throw a assertion error:

> Expression: m_matches.size() == expected

I am using CryHook5, maybe here is not the right place to ask. sorry.
Is there any way to fix it?

**Comment by QingLeiLi** [comment]
I rollback the `FC_m64.dll` file solved this problem, but after that, game crashed.

**Comment by QingLeiLi** [comment]
just hang up after this:

> 2019-06-20 01:34:10 [17340] [DEBUG] D3d11Render.cpp (369) : D3D11 device created

**Comment by otse** [comment]
> Yes you should put dawnhook together with the ultimate asi loader in the games directory. Keep in mind, that dawnhook needs to have the extension ".asi" in order to get recognized by the asi loader.

I have build DawnHook with vs 2017, and put it next to d8input.dll, and have renamed it to DawnHook.asi, but there is no sign of anything happening.

Edit: Sorry, I had to get Asi Loader x64. It works now, but I have a cracked version, and there is no gui showing.

**Comment by otse** [comment]
Do I have to press a button to make it appear?

Also, the console window starts twice now.

**Comment by BlackOfWorld** [comment]
otse, duplicate of #2

**Comment by otse** [comment]
> otse, duplicate of #2

What? I made that issue after I said that. Could you please stop moderating my posts?
